docs/usage.rst

```diff
@@ -106,5 +98,59 @@ If this is not considered, resulting intensity gradients (e.g., in the case of s
-See the description of ``tedana``'s '`:doc: approach <\approach>` for more details
+See the description of ``tedana``'s :doc:`approach <\approach>` for more details
 on how T2* values are calculated.
+
+Support and communication
+-------------------------
+All bugs, concerns and enhancement requests for this software can be submitted here: https://github.com/ME-ICA/tedana/issues.
+
+If you would like to ask a question about usage or tedana's outputs, please submit a question to `NeuroStars`_ with the ``multi-echo`` tag.
+
+All previous tedana-related questions are available under the `multi-echo tag`_.
+
+We will also attempt to archive certain common questions and associate answers in the Frequently Asked Questions (FAQ) section below.
+
+.. _multi-echo tag: https://neurostars.org/tags/multi-echo
+
+FAQ
+---
+
+ICA has failed to converge.
+```````````````````````````
+The TEDICA step may fail to converge if TEDPCA is either too strict
+(i.e., there are too few components) or too lenient (there are too many).
+
+In our experience, this may happen when preprocessing has not been applied to
+the data, or when improper steps have been applied to the data (e.g., distortion
+correction, rescaling, nuisance regression).
+If you are confident that your data have been preprocessed correctly prior to
+applying tedana, and you encounter this problem, please submit a question to `NeuroStars`_.
+
+
+I think that some BOLD ICA components have been misclassified as noise.
+```````````````````````````````````````````````````````````````````````
+``tedana`` allows users to manually specify accepted components when calling the pipeline.
+You can use the ``--manacc`` argument to specify the indices of components to accept.
+
+However, if you feel that ``tedana`` is consistently misclassifying certain BOLD-related patterns, please feel free to submit a question to `NeuroStars`_.
+We may request access to relevant visual reports, metadata, and even the input data in order to figure out what's happening.
+We are always looking for ways to improve ``tedana``'s component selection methods, and having access to data with consistent issues may help us do so.
+
+
+Why isn't v3.2 of the component selection algorithm supported in ``tedana``?
+````````````````````````````````````````````````````````````````````````````
+There is a lot of solid logic behind the updated version of the TEDICA component
+selection algorithm, first added to the original ME-ICA codebase `here`_ by Dr. Prantik Kundu.
+However, we (the ``tedana`` developers) have encountered certain difficulties
+with this method (e.g., misclassified components) and the method itself has yet
+to be validated in any papers, posters, etc., which is why we have chosen to archive
+the v3.2 code, with the goal of revisiting it when ``tedana`` is more stable.
+
+Anyone interested in using v3.2 may compile and install an earlier release of ``tedana``.
```

Comment:
Should we specify any release numbers, here ?

Reply:
Any release from 0.0.4 or earlier should work, right? I'll add that info.